cognite/neat/rules/models/_rules/_types/_base.py

```diff
@@ -222,6 +227,60 @@ def as_id(
         return ViewId(space=space, external_id=external_id, version=version)
 
 
+class ViewPropEntity(ViewEntity):
```

Comment:
if this was made to figure out which properties are "reversed" , maybe reusing methods from `_analysis` would be useful:
- get_class_linkage, to get class linkage
- get_symmetric_pairs got get which classes are linked with symmetric/reverse properties